securedrop_client/gui/login_dialog.py

```diff
@@ -35,12 +35,8 @@
 )
 
 from securedrop_client import __version__ as sd_version
-from securedrop_client.gui.widgets import (
-    LoginErrorBar,
-    LoginOfflineLink,
-    PasswordEdit,
-    SignInButton,
-)
+from securedrop_client.gui.auth import LoginOfflineLink
```

Comment:
As previously (see https://github.com/freedomofpress/securedrop-client/pull/1379/files#r771878542) here is agin the clue that `LoginDialog` belongs to `auth`.

Reply:
Will become the following once #1379 and #1380 are merged:
```diff
- from securedrop_client.gui.auth import LoginErrorBar
+ from securedrop_client.gui.auth import LoginErrorBar, LoginOfflineLink, SignInButton
```